packages/api/src/db/comment.ts

```diff
@@ -88,15 +90,10 @@ export interface GetCommentsArgs {
   readonly order: SortOrder
 }
 
-export interface GetCommentsForItemByIDArgs {
-  itemID?: readonly string[]
-}
-
-export type OptionalComment = Comment | null
 export type OptionalPublicComment = PublicComment | null
 
 export interface DBCommentAdapter {
-  createComment(args: CreateCommentArgs): Promise<PublicComment>
+  addPublicComment(args: AddPublicCommentArgs): Promise<PublicComment>
   getComments(args: GetCommentsArgs): Promise<ConnectionResult<Comment>>
-  getCommentsForItemByID(ids: readonly string[]): Promise<PublicComment[]>
+  getCommentsForItemByID(id: readonly string[]): Promise<PublicComment[]>
```

Comment:
if it's just **one** id then the type should be `string` and not `string[]`

Reply:
in the `api/src/context.ts`, this parameter is of type `readonly string[ ]` that's why doing it like this

Reply:
fixed